build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'spring-boot'
 apply plugin: 'groovy'
 apply plugin: 'signing'
 
-version = '2.0.1'
+version = '0.7.1.2'
 group = 'com.bericotech'
```

Comment:
Agreed.  We need to figure out the group.  I own a couple of domains, including "prostack.io".  That's one possibility.  However, I'm not inclined to buy one specific to Fallwizard!